### ISSUE TITLE
Accessibility: Multiple Non-UM Users header tags

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#644]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Multiple Non-UM Users

Issue notes:
"Add Multiple Users Through CSV" Should be h2 not h3

Criterion:
2.4.10 Section Headings
